controls/V-72049.rb

```diff
@@ -45,19 +51,51 @@
 environment variables."
   tag "fix_id": "F-78401r1_fix"
 
-  # @todo - test for values more restrictive than 077
-  file_lines = command('grep -i -s umask /home/*/.*').stdout.split("\n")
-  if !file_lines.nil? and !file_lines.empty?
-    file_lines.each do |curr_line|
-      file_name = curr_line.split(':').first
-      describe command("grep -i umask #{file_name}") do
-        its('stdout.strip') { should match %r{^umask\s+.*077} }
-      end
+  # Get all interactive users
+  ignore_shells = non_interactive_shells.join('|')
+  
+  # Get home directory for users with UID >= 1000 or UID == 0 and support interactive logins.                                                              
+  findings = Set[]
+  dotfiles = Set[]
+  umasks = {}
+  interactive_users = users.where{ !shell.match(ignore_shells) && (uid >= 1000 || uid == 0)}.entries
```

Comment:
1000 should be pulled from `/etc/login.defs` `UID_MIN`

Reply:
I made this change.

Reply:
@djhaynes Did the code get pushed up? I'm not seeing where this is getting pulled.

Reply:
The change should be in there now.